lighthouse-core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -0,0 +1,444 @@
+/**
```

Comment:
is bisect going to be preserved for any of the things in here?

Reply:
not really, the file was moved from `trace-of-tab` so that history should be there but since I indented everything the blame will be pretty much wiped out